src/ml_chooser.cpp

```diff
@@ -5,6 +5,7 @@ namespace myrps
   MLChooser::MLChooser(int rounds_per_match)
   {
     this->n = 5; // default value. must be greater than 1
+    this->file_writer = FileWriter("ml_data_" + to_string(n) + ".csv", "ml_data");
```

Comment:
```suggestion
    this->file_writer = FileWriter("ml_data_" + to_string(n) + ".csv", "data/ml_data");
```

Reply:
an empty `ml_data` directory is made and unused at top level of the project without this correction.